documentation/faqs/index.markdown

```diff
@@ -19,15 +19,17 @@ _What operating systems does MoveIt! work with?_
 
 _What is the difference between MoveIt! and ROS?_
 
-  * ROS (Robot Operating System) is an open-source meta-operating system for robots that provides low-level functionality like a build system, message passing, device drivers and some integrated capabilities like navigation. MoveIt! provides functionality for kinematics, motion/path planning, collision checking, 3D perception, robot interaction and much, much more. MoveIt! is a primary source of a lot of the functionality for manipulation (and mobile manipulation) in ROS. MoveIt! builds on the ROS messaging and build systems and utilizes some of the common tools in ROS like the ROS Visualizer (Rviz) and the ROS robot format (URDF). MoveIt! is fast becoming the entry point into ROS, especially through the use of the MoveIt! Setup Assistant for configuring new robots.
+  * MoveIt! runs on top of ROS (Robot Operating System).
+  * ROS is an open-source meta-operating system for robots that provides low-level functionality like a build system, message passing, device drivers and some integrated capabilities like navigation. MoveIt! provides functionality for kinematics, motion/path planning, collision checking, 3D perception, robot interaction and much, much more. MoveIt! is a primary source of a lot of the functionality for manipulation (and mobile manipulation) in ROS. MoveIt! builds on the ROS messaging and build systems and utilizes some of the common tools in ROS like the ROS Visualizer (Rviz) and the ROS robot format (URDF). MoveIt! is fast becoming the entry point into ROS, especially through the use of the MoveIt! Setup Assistant for configuring new robots.
 
 _What is the difference between MoveIt! and Arm Navigation?_
 
   * Arm Navigation was the predecessor to MoveIt!. MoveIt! is significantly different to Arm Navigation in system architecture and implementation, especially in terms of performance and extensibility. MoveIt! moved away from the process based architecture that Arm Navigation used, mainly to lower the cost of maintaining a representation of the world in multiple nodes. MoveIt! also takes advantage of the parallelization provided by collision checkers like FCL.
 
 _What is the difference between MoveIt! and ROS-Industrial?_
 
-  * The ROS-Industrial consortium aims to use the advanced tools in MoveIt! and ROS to build and demonstrate advanced applications in industry. They were inspired by the success of Arm Navigation (MoveIt!'s predecessor) on the PR2 robot. ROS-Industrial also develops and maintains ROS drivers for industrial robots (using the PR2 drivers as a template in several cases).
+  * The ROS-Industrial consortium aims to use the advanced tools in MoveIt! and ROS to build and demonstrate advanced applications in industry. They were inspired by the success of Arm Navigation (MoveIt!'s predecessor) on the PR2 robot. 
+  * ROS-Industrial also develops and maintains ROS drivers for industrial robots (using the PR2 drivers as a template in several cases).
```

Comment:
Interesting.

Would it then perhaps be an idea to change this section slightly, and make it about the *relationship* between MoveIt, arm nav and ROS-Industrial?

Talking about differences and then not including something about single technology / package vs project (which makes use of that technology) seems strange to me. It implicitly places MoveIt, ROS-Industrial and arm_navigation on the same conceptual level (for me at least), which does not seem entirely correct.

The relationship between all three is clear though: MoveIt is the successor to arm_navigation, and ROS-Industrial, a project making use of various technologies in ROS, uses MoveIt to solve some of its motion planning problems.

I'll see if I can provide a PR.